grunttasks/rev.js

```diff
@@ -14,6 +14,8 @@ module.exports = function (grunt) {
           '<%= yeoman.dist %>/scripts/{,*/}*.js',
           '<%= yeoman.dist %>/styles/{,*/}*.css',
           '<%= yeoman.dist %>/images/{,*/}*.{png,jpg,jpeg,gif,webp,svg}',
+          '!<%= yeoman.dist %>/images/apple_app_store_button/{,*/}*.{png,jpg,jpeg,gif,webp,svg}',
+          '!<%= yeoman.dist %>/images/google_play_store_button/{,*/}*.{png,jpg,jpeg,gif,webp,svg}',
```

Comment:
I think so? These images are inline (in `<img>` tags), which @shane-tomlinson and I weren't sure how to make play nice with hashed file names. The images should only ever be served to each user once, so caching doesn't seem to be of huge benefit.

I'm open to caching if it's easy to incorporate with inline images.